covalent_data/app/api/api_v0/endpoints/fs.py

```diff
@@ -46,6 +47,10 @@
         secure=MINIO_USE_TLS,
     )
     backend = MinioStorageBackend(minio_client, settings.FS_STORAGE_BUCKET)
+elif settings.FS_STORAGE_BACKEND == "s3":
+    # Reads AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_DEFAULT_REGION from environment vars
+    s3_client = client = boto3.client("s3")
```

Comment:
Is there a reason for `s3_client = client = ...`?

Reply:
ah yes good catch, that's a redundant variable assignment, forgot to remove this when I was doing some refactoring,  will fix 👍